WordPress/src/main/java/org/wordpress/android/ui/stats/StatsActivity.java

```diff
@@ -696,66 +683,10 @@ private void stopStatsService() {
         }
     }
 
-    /*
-     * receiver for broadcast from StatsService which alerts when stats update has started/ended
-     */
-    private final BroadcastReceiver mReceiver = new BroadcastReceiver() {
-        @Override
-        public void onReceive(Context context, Intent intent) {
-            String action = StringUtils.notNullStr(intent.getAction());
-
-            if (action.equals(StatsService.ACTION_STATS_UPDATING)) {
```

Comment:
Yes, EXTRA_IS_ERROR was never set, and the code in Stats activity that checks for it was redundant.